membership-attribute-service/app/controllers/HealthCheckController.scala

```diff
@@ -16,14 +17,25 @@ class BoolTest(name: String, exec: () => Boolean) extends Test {
   override def ok = exec()
 }
 
-class HealthCheckController extends Results {
-  lazy val zuora = NormalTouchpointComponents.zuoraService
-  lazy val salesforce = NormalTouchpointComponents.contactRepo.salesforce
+class HealthCheckController extends Results with LazyLogging {
 
-  val tests: Seq[Test] = Seq(
-    new BoolTest("ZuoraPing", () => zuora.lastPingTimeWithin(2.minutes)),
-    new BoolTest("Salesforce", () => salesforce.isAuthenticated)
-  )
+  private lazy val wrappedZuoraService = new HealthCheckableService {
+    private val duration = 2.minutes
+    override def serviceName = "ZuoraService"
+    override def checkHealth: Boolean =
+      if (zuoraService.lastPingTimeWithin(duration)) {
```

Comment:
I notice that the lastPingTimeWithin function (in the depths of membership-common) is making use of a ScheduledTask (https://github.com/guardian/membership-common/blob/master/src/main/scala/com/gu/zuora/soap/Client.scala#L91). 

Could we add this logging at that level (and pick it up for free in all the other apps which use this task), and take Zuora out of the healthcheck altogether?

Reply:
See https://github.com/guardian/membership-common/pull/551